addons/l10n_sa_edi/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models, fields, api, _
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    l10n_sa_api_mode = fields.Selection(related='company_id.l10n_sa_api_mode', readonly=False)
+
+    @api.depends('company_id')
+    def _compute_company_informations(self):
+        super()._compute_company_informations()
+        for record in self:
+            record.company_informations += _('\nBuilding Number: %s, Plot Identification: %s \nNeighborhood: %s') % (self.company_id.l10n_sa_edi_building_number, self.company_id.l10n_sa_edi_plot_identification, self.company_id.l10n_sa_edi_neighborhood)
```

Comment:
we have never done this before in other localizations...  (it might just be wrong for non SA companies.  Maybe an if might do)

Reply:
@jco-odoo You're right, I checked again and it's not used in any other localization. I suggest we just remove it

Reply:
@mebe-odoo You could be right to add it.  It is just that it should only be done for SA companies.  (country_code == 'SA')

Reply:
Done